.circleci/check-testnet-deployment.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+const { waitUntilSync, getBlockHeight, getCommit } = require('@orbs-network/orbs-nebula/lib/metrics');
```

Comment:
We decided to unpublish `@orbs-network/orbs-nebula` a month ago. Why is it used here?

Reply:
In your previous review you wanted me to use your code from Nebula
This is the official package. We talked about unpublish (and we did) the other package (`orbs-nebula`)

This scoped package is the official one which we left

Reply:
Official docs still use `npm install -g orbs-nebula`

Also maybe it would be easier to just install from our github master? We rarely release new nebula version.